### ISSUE TITLE
WCAG issue - Analytics choice in Actions menu on Media page disappears at high zoom levels

### ISSUE THREAD

**Issue by judyucb** [Kaltura-EDU/issues-tracking#28]
Similar to https://github.com/Kaltura-EDU/issues-tracking/issues/12, I've opened up Cases referencing WCAG 1.4.4 for the Media Page.
One in 2021 (#00361540), one in 2023 (#00478875), one in 2024 (#00515758). 

On the Media page, however, the issue has only been items disappearing from the Actions menu when Zooming in. One of the disappearances (Edit item), which they originally told me was a "cosmetic decision, as the design is adversely affected under a certain width", was eventually resolved in ADA-1303 (i.e. "Media page - Edit action is now available in 400% zoom", https://knowledge.kaltura.com/help/kms-kaf-release-notes#kms-version-5129x-july-2-2024). 

Actions>Analytics on Media page still disappears, however, when Zoomed in to only 175%, and rationale I've been given is similar: "the analytics feature is not supported at higher zoom levels." i.e. it sounds like the limitation isn't necessarily because of screen real estate on the page on which the item is missing (Media page), it's that they can't figure out how to get the target page (Analytics) to work zoomed in.
